backend/fleet_management/models.py

```diff
@@ -11,6 +11,9 @@ def __str__(self):
 
 class Car(models.Model):
     plates = models.CharField(max_length=10, blank=False, unique=True)
+    description = models.CharField(max_length=500, blank=True)
+    mileage_unit = models.CharField(max_length=10, blank=False)
```

Comment:
This field needs choices https://docs.djangoproject.com/en/2.1/ref/models/fields/#choices